src/controllers/CheckYourAppealController.ts

```diff
@@ -46,6 +46,33 @@ export class CheckYourAppealController extends BaseAsyncHttpController {
             .chain(data => Maybe.fromNullable(data[APPEALS_KEY]))
             .extract() as Appeal;
 
+        const regionPrefix: string = appealsData.penaltyIdentifier.companyNumber.slice(0,2);
```

Comment:
That logic should be extracted and unit tested.

Reply:
Do we slice the companyNumber even when it doesn't contain any letters?